homeassistant/components/simplisafe/lock.py

```diff
@@ -37,36 +34,60 @@
 class SimpliSafeLock(SimpliSafeEntity, LockDevice):
     """Define a SimpliSafe lock."""
 
-    def __init__(self, system, lock):
+    def __init__(self, simplisafe, system, lock):
         """Initialize."""
-        super().__init__(system, lock.name, serial=lock.serial)
+        super().__init__(simplisafe, system, lock.name, serial=lock.serial)
         self._lock = lock
 
     @property
     def is_locked(self):
         """Return true if the lock is locked."""
-        return STATE_MAP.get(self._lock.state) == STATE_LOCKED
+        return self._state == STATE_LOCKED
```

Comment:
There is no reason to actually use `STATE_LOCKED` and `STATE_UNLOCKED`. You can also use a boolean. Those state constants are an implementation detail of the lock entity. You just need to return a boolean from `is_locked`